py-polars/polars/dataframe/frame.py

```diff
@@ -2276,7 +2277,16 @@ def to_pandas(  # noqa: D417
             )
 
         date_as_object = kwargs.pop("date_as_object", False)
-        return tbl.to_pandas(date_as_object=date_as_object, **kwargs)
+        with warnings.catch_warnings():
+            # Needs fixing upstream in pyarrow
+            # Silence here as it's something Polars users can't do
+            # anything about.
+            warnings.filterwarnings(
+                "ignore",
+                message="make_block is deprecated and will be removed",
+                category=DeprecationWarning,
+            )
```

Comment:
I think you can use `filterwarnings` as a context manager directly, right?
```suggestion
        # Needs fixing upstream in pyarrow
        # Silence here as it's something Polars users can't do
        # anything about.
        with warnings.filterwarnings(
                "ignore",
                message="make_block is deprecated and will be removed",
                category=DeprecationWarning,
            ):
```

Reply:
doesn't work for me, I get
```
E   AttributeError: __enter__
```

Reply:
I was mistaken, sorry for the misdirection!